include/aikido/planner/parabolic/ParabolicSmoother.hpp

```diff
@@ -129,6 +130,60 @@ std::unique_ptr<trajectory::Spline> doShortcutAndBlend(
     double _checkResolution = DEFAULT_CHECK_RESOLUTION,
     double _tolerance = DEFAULT_TOLERANCE);
 
+/// Class for performing parabolic smoothing on trajectories
+class SmoothTrajectoryPostProcessor
+    : public aikido::planner::TrajectoryPostProcessor
+{
+public:
+  /// \param _velocityLimits maximum velocity for each dimension.
+  /// \param _accelerationLimits maximum acceleration for each dimension.
+  /// \param _collisionTestable Check whether a position is feasible.
+  /// \param _enableShortcut Whether shortcutting is used in smoothing.
+  /// \param _enableBlend Whether blending is used in smoothing.
+  /// \param _shortcutTimelimit Timelimit for shortcutting. It is ineffective
+  /// when _enableShortcut is false.
+  /// \param _blendRadius Blend radius for blending. It is ineffective
+  /// when _enableBlend is false.
+  /// \param _blendIterations Blend iterations for blending. It is
+  /// ineffective when _enableBlend is false.
+  /// \param _smootherFeasibilityCheckResolution the resolution in discretizing
+  /// a segment in checking the feasibility of the segment.
+  /// \param _smootherFeasibilityApproxTolerance this tolerance is used in a
+  /// piecewise linear discretization that deviates no more than
+  /// \c _smootherFeasibilityApproxTolerance from the parabolic ramp along any
+  /// axis, and then checks for configuration and segment feasibility along that
+  /// piecewise linear path.
+  SmoothTrajectoryPostProcessor(
+      const Eigen::VectorXd& _velocityLimits,
+      const Eigen::VectorXd& _accelerationLimits,
+      const aikido::constraint::TestablePtr& _collisionTestable,
+      bool _enableShortcut = true,
+      bool _enableBlend = true,
+      double _shortcutTimelimit = DEFAULT_TIMELIMT,
+      double _blendRadius = DEFAULT_BLEND_RADIUS,
+      int _blendIterations = DEFAULT_BLEND_ITERATIONS,
+      double _smootherFeasibilityCheckResolution = DEFAULT_CHECK_RESOLUTION,
```

Comment:
I don't think "smoother" is necessary in this variable name (and the one below).